workflow_tests/lint.test.ts

```
@@ -91,32 +103,38 @@ describe('test workflow lint', () => {
                 action: 'opened',
             };
             it('runs the lint', async () => {
-                const repoPath = mockGithub.repo.getPath('testLintWorkflowRepo') || '';
+                const repoPath = mockGithub.repo.getPath('testLintWorkflowRepo') ?? '';
                 const workflowPath = path.join(repoPath, '.github', 'workflows', 'lint.yml');
                 let act = new ExtendedAct(repoPath, workflowPath);
                 act = utils.setUpActParams(act, event, eventOptions, {}, githubToken);
                 const testMockSteps = {
                     lint: mocks.LINT__LINT__STEP_MOCKS,
                 };
-                const result = await act.runEvent(event, {
-                    workflowFile: path.join(repoPath, '.github', 'workflows', 'lint.yml'),
-                    mockSteps: testMockSteps,
-                    actor,
-                    logFile: utils.getLogFilePath('lint', expect.getState().currentTestName),
-                });
 
-                assertions.assertLintJobExecuted(result);
+                try {
+                    const result = await act.runEvent(event, {
+                        workflowFile: path.join(repoPath, '.github', 'workflows', 'lint.yml'),
+                        mockSteps: testMockSteps,
+                        actor,
+                        logFile: utils.getLogFilePath('lint', expect.getState().currentTestName),
+                    });
+
+                    assertions.assertLintJobExecuted(result);
+                } catch (error) {
+                    console.error(error);
+                }
```

Why do we suppress the error here?